packages/flutter/lib/src/widgets/_window_positioner.dart

```diff
@@ -0,0 +1,510 @@
+import 'dart:ui';
+import 'package:flutter/foundation.dart';
+import '_window.dart';
+
+/// Defines how a child window will be placed relative to the anchor rectangle
+/// of its parent.
+///
+/// The specified anchor is used to derive an anchor point on the anchor rectangle that
+/// the child [BaseWindowController] will be positioned relative to.
+/// If a corner anchor is set (e.g. [topLeft] or [bottomRight]),
+/// the anchor point will be at the specified corner; otherwise, the derived anchor point
+/// will be centered on the specified edge, or in the center of the anchor rectangle
+/// if no edge is specified.
+///
+/// {@macro flutter.widgets.windowing.experimental}
+@internal
+enum WindowPositionerAnchor {
```

Comment:
Could [Alignment](https://main-api.flutter.dev/flutter/painting/Alignment-class.html) work for this? Or if not, maybe this should similarly wrap an x and y double instead of just being an enum.